android/app/src/main/java/network/perun/app/MainActivity.java

```diff
@@ -94,8 +96,22 @@ public Node(Config cfg, Wallet wallet) throws Exception {
         }).start();
     }
 
+    public void reconnect() throws Exception {
+        Context ctx = Prnm.contextWithTimeout(600);
+        try {
+            client.reconnect(ctx);
+        } finally {
+            ctx.cancel();
+        }
+    }
+
     public void close() {
-        client.close();
+        try
+        {
```

Comment:
you never insert a newline before a `{`, why here?

Reply:
Normally the formatter does this, I did not write this change in Android Studio :laughing: